vendor/mpdf/mpdf/src/Mpdf.php

```diff
@@ -962,48 +964,49 @@ public function __construct(array $config = [])
 		$originalConfig = $config;
 		$config = $this->initConfig($originalConfig);
 
-		$this->sizeConverter = new $this->SizeConverterClass($this->dpi, $this->default_font_size);
+		$this->sizeConverter = new SizeConverter($this->dpi, $this->default_font_size);
 
-		$this->colorModeConverter = new $this->ColorModeConverterClass();
-		$this->colorSpaceRestrictor = new $this->ColorSpaceRestrictorClass(
+		$this->colorModeConverter = new ColorModeConverter();
+		$this->colorSpaceRestrictor = new ColorSpaceRestrictor(
 			$this,
 			$this->colorModeConverter,
 			$this->restrictColorSpace
 		);
-		$this->colorConverter = new $this->ColorConverterClass($this, $this->colorModeConverter, $this->colorSpaceRestrictor);
+		$this->colorConverter = new ColorConverter($this, $this->colorModeConverter, $this->colorSpaceRestrictor);
 
 
-		$this->gradient = new $this->GradientClass($this, $this->sizeConverter, $this->colorConverter);
-		$this->tableOfContents = new $this->TableOfContentsClass($this, $this->sizeConverter);
+		$this->gradient = new Gradient($this, $this->sizeConverter, $this->colorConverter);
+		$this->tableOfContents = new TableOfContents($this, $this->sizeConverter);
 
-		$this->cache = new $this->CacheClass($config['tempDir']);
-		$this->fontCache = new $this->FontCacheClass(new $this->CacheClass($config['tempDir'] . '/ttfontdata'));
+		$this->cache = new Cache($config['tempDir']);
+		$this->fontCache = new FontCache(new Cache($config['tempDir'] . '/ttfontdata'));
 
-		$this->fontFileFinder = new $this->FontFileFinderClass($config['fontDir']);
+		$this->fontFileFinder = new FontFileFinder($config['fontDir']);
 
-		$this->cssManager = new $this->CssManagerClass($this, $this->cache, $this->sizeConverter, $this->colorConverter);
+		$this->cssManager = new CssManager($this, $this->cache, $this->sizeConverter, $this->colorConverter);
 
-		$this->otl = new $this->OtlClass($this, $this->fontCache);
+		$this->otl = new Otl($this, $this->fontCache);
 
-		$this->form = new $this->FormClass($this, $this->otl, $this->colorConverter);
+		$this->form = new Form($this, $this->otl, $this->colorConverter);
 
-		$this->hyphenator = new $this->HyphenatorClass($this);
+		$this->hyphenator = new Hyphenator($this);
 
-		$this->logger = new $this->NullLoggerClass();
+		$this->logger = new NullLogger();
 
-		$this->imageProcessor = new $this->ImageProcessorClass(
+		$this->imageProcessor = new ImageProcessor(
```

Comment:
In previous changes (0576bb86cc46a99f745f474945f8e54e5b0a41ba) you modified the ImageProcessor. I guess you undo here these changes.
If I have an image in my wiki article (e.g. `{{:file.jpg?400|}}`), it is not exported, but appears as a red cross icon.

Reply:
Ah, yes. I now remember I tried a somewhat more general solution, that was supposed to prevent this kind of problem during updates of the upstream library, but it was declined (mpdf/mpdf#460).

I'll fix this problem locally and we have to remember to do that for every update of the upstream library or find enough time to fix it properly.